chapters/sampling-randomization-power.tex

```diff
@@ -1,266 +1,611 @@
-
-\section{Sampling and randomization}
-
-The sampling and randomization processes we choose
-play an important role in determining the size of the confidence intervals
-for any estimates generated from that sample,
-and therefore our ability to draw conclusions.
-Random sampling and random assignment serve different purposes.
-Random \textit{sampling} ensures that you have unbiased population estimates,
-and random \textit{assignment} ensures that you have unbiased treatment estimates.
-If you randomly sample or assign a set number of observations from a set frame,
-there are a large -- but fixed -- number of permutations which you may draw.
-
-In reality, you have to work with exactly one of them,
-so we put a lot of effort into making sure that one is a good one
-by reducing the probability of the data indicating that results or correlations are true when they are not.
-In large studies, we can use what are called \textbf{asymptotic standard errors}\sidenote{
-  \url{https://stats.stackexchange.com/q/88491}}
-to express how far away from the true population parameters our estimates are likely to be.
-These standard errors can be calculated with only two datapoints:
-the sample size and the standard deviation of the value in the chosen sample.
-They are also typically the best case scenario for the population given the data structure.
-In small studies, such as those we often see in development,
-we have to be much more careful, particularly about practical considerations
-such as determining the representative population
-and fitting any constraints on study and sample design.
-This section introduces universal basic principles for sampling and randomization.
-
-\subsection{Sampling}
+\subsection{Randomized sampling}
 
+% sampling universe: the master dataset
 \textbf{Sampling} is the process of randomly selecting units of observation
 from a master list of individuals to be included in data collection.\sidenote{
-  \url{https://dimewiki.worldbank.org/Sampling_\%26_Power_Calculations}}
-  \index{sampling}
+	\url{https://dimewiki.worldbank.org/Sample\_Size\_and\_Power\_Calculations}}
+\index{sampling}
 That master list may be called a \textbf{sampling universe}, a \textbf{listing frame}, or something similar.
 We recommend that this list be organized in a \textbf{master dataset}\sidenote{
-  \url{https://dimewiki.worldbank.org/Master_Data_Set}},
+	\url{https://dimewiki.worldbank.org/Master\_Data\_Set}},
 creating an authoritative source for the existence and fixed
 characteristics of each of the units that may be surveyed.\sidenote{
-	\url{https://dimewiki.worldbank.org/Unit_of_Observation}}
+	\url{https://dimewiki.worldbank.org/Unit\_of\_Observation}}
 The master dataset indicates how many individuals are eligible for data collection,
-and therefore contains statistical information about the likelihood that each will be chosen.
+and therefore contains statistical information about
+the likelihood that each will be chosen.
```

Comment:
this content should be moved to master dataset section that comes earlier

Reply:
agreed, will do.

Reply:
Moved in 60c5987aaab34c88162824aac60b5db6d4f64b69